packages/contracts/contracts/maci/Tally.sol

```diff
@@ -62,15 +59,15 @@ contract Tally is TallyBase, IPayoutStrategy, Pausable {
   event ResultAdded(uint256 indexed index, uint256 indexed result);
 
   /// @notice custom errors
-  error CooldownPeriodNotOver();
   error InvalidBudget();
   error NoProjectHasMoreThanOneVote();
-  error InvalidWithdrawal();
   error AlreadyInitialized();
   error NotInitialized();
   error NotCompletedResults();
   error TooManyResults();
   error AlreadyClaimed();
+  error DepositWindowClosed();
+  error DepositWindowNotClosed();
```

Comment:
Can we add tests for all new logic? 

Making sure we cover all branches and also both cases for `<=`